assets/dolphin/external/manifest.txt

```diff
@@ -140,3 +140,10 @@ Max butthurt: 10
 Min level: 3
 Max level: 3
 Weight: 3
+
+Name: L1_Sleigh_ride_128x64
+2Min butthurt: 0
+3Max butthurt: 14
+4Min level: 1
+5Max level: 3
+6Weight: 4
```

Comment:
Hi, there is a problem here, these numbers at line starts should not be here :)

Reply:
Oops, you're absolutely right